libs/core/src/lib/icon-tab-bar/directives/dnd/icon-bar-dnd-list.directive.ts

```diff
@@ -0,0 +1,97 @@
+import { AfterViewInit, Directive, ElementRef, Input, OnDestroy, Optional } from '@angular/core';
+import { ReplaySubject, Subject } from 'rxjs';
+
+import { DragDrop, DragRef, DropListRef } from '@angular/cdk/drag-drop';
+import { IconBarDndItemDirective } from './icon-bar-dnd-item.directive';
+import { debounceTime, takeUntil } from 'rxjs/operators';
+import { IconBarDndContainerDirective } from './icon-bar-dnd-container.directive';
+
+@Directive({
+    selector: '[fdIconBarDndList]',
```

Comment:
Here is an interesting one, somewhere we have directives in camelCase and somewhere else in kebab-case, which one is preferred? 

@InnaAtanasova @droshev do we have strict rules on that? I can't remember.

Reply:
we usually have the kebab-case and disable tslint for the `directive-selector`. For example:
```
@Directive({
    // tslint:disable-next-line:directive-selector
    selector: '[fd-avatar-group-focusable-avatar]',
    host: { class: 'fd-avatar-group__focusable-avatar' }
})
```

Reply:
But you can add both since we don't have a clear guidelines (for now):
```
[fdIconBarDndList], [fd-icon-bar-dnd-list]
```

Reply:
So let's add the second, kebab-case variant also.